.gitignore

```diff
@@ -193,3 +193,6 @@ $RECYCLE.BIN/
 *.lnk
 
 # End of https://www.toptal.com/developers/gitignore/api/node,macos,windows,visualstudiocode
+
+# Template development directory
+template-dev/
```

Comment:
I added this so it's easy to test locally and generate the template, can remove but maybe it's ok to leave it 🤷‍♂️

Reply:
Okay to leave it I think